.github/workflows/ci.yml

```diff
@@ -205,3 +205,19 @@ jobs:
       run: |
         source $HOME/emsdk/emsdk_env.sh
         ./scripts/emcc-tests.sh
+
+  # Windows + gcc needs work before the tests will run, so just test the compile
+  build-mingw:
+    name: mingw
+    runs-on: windows-latest
+    steps:
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - uses: actions/checkout@v1
+    - name: cmake
+      run: |
+        mkdir -p out
+        cmake -S . -B out -G "MSYS Makefiles"
+    - name: build
+      run: cmake --build out
```

Comment:
Shouldn't this run in the mingw shell?

Reply:
The MSYS Makefiles seem to pick up the correct GNU compiler. I tested using `shell: bash` and there was no difference.